### PR TITLE
Mm/sm 753: mocks not being added to libraries-state.json

### PR DESCRIPTION
**Pull request by MarcMcIntosh**
## Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

Removes the name property from VariatonMock as it conflicts with the out put form prismic-mocks library
## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [x] All new and existing tests are passing.

### CODE REVIEW

File: packages/core/src/models/Slice.ts [hunk 1]
```diff
@@ -30,13 +30,25 @@ export const VariationSM = t.intersection([
...
+// });
+
+export const VariationMock = t.intersection([
+  t.type({
+    variation: t.string,
+    slice_type: t.string,
+    items: t.array(t.unknown),
+    primary: t.record(t.string, t.unknown),
+  }),
+  t.partial({
```
Comment (MarcMcIntosh) [inline packages/core/src/models/Slice.ts]: Not sure this is right, as the name seems to be important

File: cypress/integration/slices/00-create.spec.js [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
   const editedSliceName = "TestSlice2";
   const lib = "slices--ecommerce"; // name of the first lib of the next project.
-  const path = "e2e-projects/next/slices/ecommerce";
-  const generatedPath =
-    "e2e-projects/next/.slicemachine/assets/slices/ecommerce"; // path to th library
+  const pathToLib = path.join("e2e-projects", "next", "slices", "ecommerce");
+  const generatedPath = path.join("e2e-projects", "next", ".slicemachine", "assets", "slices", "ecommerce"); // path to th library
+  const pathToLibraryState = path.join("e2e-projects", "next", ".slicemachine", "libraries-state.json")
+
+  const pathToMock = (slice) => path.join("e2e-projects", "next", ".slicemachine", "assets", "slices", "ecommerce", slice, "mocks.json")
```
Comment (chengoconnell) [inline cypress/integration/slices/00-create.spec.js]: nice!

File: packages/slice-machine/lib/mock/Slice.ts [hunk 3]
```diff
@@ -74,7 +74,10 @@ export default function MockSlice(
     }),
     type: SlicesTypes.SharedSlice,
   };
-  return sliceMockConfig.map((sc) =>
-    generateSliceMock(sliceModel, sc)(renderSliceMock)
-  );
+  // console.dir({legacyMockConfig, model, sliceMockConfig, sliceModel}, {depth: null})
```
Comment (chengoconnell) [inline packages/slice-machine/lib/mock/Slice.ts]: are these comments left for future devs?

Comment (MarcMcIntosh) [reply packages/slice-machine/lib/mock/Slice.ts]: nope, I just forgot to remove them :)

Comment (MarcMcIntosh) [reply packages/slice-machine/lib/mock/Slice.ts]: ..looks like i did remove them ?

File: packages/core/src/models/Slice.ts [hunk 4]
```diff
@@ -32,11 +32,11 @@ export type VariationSM = t.TypeOf<typeof VariationSM>;
 
 export const VariationMock = t.type({
   variation: t.string,
-  name: t.string,
```
Comment (chengoconnell) [inline packages/core/src/models/Slice.ts:35]: do we not need a name at all then?

Comment (MarcMcIntosh) [reply packages/core/src/models/Slice.ts:35]: Not from what I can find.
Looks like the name property on the mock was copied from the model type :)

Comment (MarcMcIntosh) [reply packages/core/src/models/Slice.ts:35]: It's also been removed in the prismic-mocks package

**Review by chengoconnell — APPROVED** [review]

**Pull request merged**